src/Find-SDC-Repositories.js

```diff
@@ -0,0 +1,119 @@
+const graphql = require("graphql-request");
+const { zipWith } = require("lodash");
+const prompt = require("prompt");
+const fs = require("fs");
+const process = require("process");
+const mkdirp = require("mkdirp");
+
+const config = {
+  file: {
+    filePrefix: "SDC-Teams-Github-Repos-",
+    directoryPath: "outputs/SDCTeamGithubRepos/"
+  },
+  api: {
+    endpoint: "https://api.github.com/graphql",
+    options: {
+      headers: {
+        Authorization: null
+      }
+    },
+    query: `
+    query { 
```

Comment:
I think we should put the query in a separate file, as it's the sort of thing that is likely to change independently of the code to invoke the Github API.

Maybe structure the folders something like this?

```
graphql
└── queries
    └── findRepoPermissionsPerTeam.js
```